schemer/__init__.py

```diff
@@ -129,6 +125,8 @@ def _verify_type(self, spec, path):
         elif not isinstance(field_type, type) and not isinstance(field_type, types.FunctionType):
             raise SchemaFormatException("Unsupported field type at {}. Type must be a type, a function, an Array or another Schema", path)
 
+    def _valid_schema_default(self, value):
+        return value == {}
```

Comment:
fi we do not restrict it, we are opening ourselves up to a situation where someone can add values that are not valid through defaults